client/state/selectors/get-inline-help-support-variation.js

```diff
@@ -0,0 +1,39 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+import { isDirectlyFailed } from 'state/selectors';
+import isHappychatAvailable from 'state/happychat/selectors/is-happychat-available';
+import isHappychatUserEligible from 'state/happychat/selectors/is-happychat-user-eligible';
+import { isTicketSupportEligible } from 'state/help/ticket/selectors';
+
+const SUPPORT_DIRECTLY = 'SUPPORT_DIRECTLY';
```

Comment:
These constants should live somewhere more generic and be sharable. That way we can refer to them and compare the result of the selector against them.

Reply:
Agreed.

Reply:
Solved by just exporting from the selector - this feels like a good pattern as the selector file is then the single source of the constants it uses.